testem.js

```diff
@@ -4,7 +4,6 @@ module.exports = {
   "disable_watching": true,
   "firefox_user_js": "./firefox_config.js",
   "launch_in_ci": [
-    "PhantomJS",
```

Comment:
Travis had failures with testing that document.focus() does result in a blur event from the active (focused) element.